tasks/main.yml

```diff
@@ -5,6 +5,13 @@
     path: /var/pca/pca-gophish-composition
     state: directory
 
+- name: Install packages needed by Ansible and gophish-tools helper scripts
+  ansible.builtin.package:
+    name:
+      - at
+      - jq
+      - tar
```

Comment:
The `ansible.builtin.package` step was moved earlier in the process to ensure that the `tar` command is installed for the proceeding step.  This is largely due to the fact that the `geerlingguy/docker-amazonlinux2023-ansible:latest` image does not come with `tar` preinstalled.  However it stands to reason that we should not assume that `tar` will always be installed by default.

Reply:
I suggest putting this command back where it was and adding a new task to install `tar` here.

Reply:
Given that this is a single package, is the preference
the First:
```
  ansible.builtin.package:
    name:
      - tar

```
or the Second:
```
  ansible.builtin.package:
    name: tar
```

Reply:
The first one is preferred, since it makes it easy to add another package should that ever prove necessary.

Reply:
@jsf9k Done.

Reply:
@jsf9k, Done.